docs/rules/string-content.md

```diff
@@ -75,11 +75,3 @@ For example, if you want to enforce `...` → `…`:
 	}
 }
 ```
-
-## Default Pattern
```

Comment:
I think we should keep this section, but rename it to `Pattern Ideas` or something, to give people some inspiration on what kind of patterns to use.

We could add `...` => `…` there too.